models/reporting/commissioning/projects/cltcs/cltcs_score_treatment_snapshot_input.sql

```
@@ -0,0 +1,54 @@
+  scored population.
+- `table_refresh_date` is the project convention for the optional
+  `updated_at` column used alongside the check strategy.
+
+Downstream:
+- `snapshots/cltcs_score_treatment_snapshot.yml`
+*/
+
+select
+      sk_patient_id
+    -- scaled subdomain scores (cohort-relative z-scores)
+    , scaled_score_biomarker_gaps
```

SQLFluff (4.2.2)

[error] Found leading comma ','. Expected only trailing near line breaks.

(LT04)